src/devices/lixee.ts

```diff
@@ -1938,10 +1996,10 @@ export const definitions: DefinitionWithExtend[] = [
             if (event.type === "start") {
                 event.data.device
                     .getEndpoint(1)
-                    .read("liXeePrivate", ["linkyMode", "currentTarif"], {manufacturerCode: null})
+                    .read("liXeeCustom", ["linkyMode", "currentTarif"], {manufacturerCode: null})
```

Comment:
This is where I see one of the failures:
```
[2025-11-09 20:13:58] warning:  zhc:lixee: Failed to read zigbee attributes during startup: Error: Cluster with name 'liXeeCustom' does not exist
```

Reply:
No we don't, should use `liXeePrivate` here. If that doesn't work, before this line do `console.log(event.data.device._customClusters)` and see if the cluster is in there.

Reply:
They seem quite similar:
https://github.com/Koenkk/zigbee-herdsman-converters/blob/9d90c31ba1cd02a4998e3c95c32c73331b186d12/src/devices/heiman.ts#L1115
https://github.com/Koenkk/zigbee-herdsman-converters/blob/9d90c31ba1cd02a4998e3c95c32c73331b186d12/src/devices/niko.ts#L312
https://github.com/Koenkk/zigbee-herdsman-converters/blob/9d90c31ba1cd02a4998e3c95c32c73331b186d12/src/devices/shelly.ts#L500

I tried to make the code look as close as possible to other cluster move PRs which I checked.

How can I make it easier for you to debug this problem?